pkgs/development/tools/misc/saleae-logic-2/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, fetchurl, appimageTools }:
+let name = "saleae-logic-2";
```

```suggestion
let
  name = "saleae-logic-2";
```
Please format the file with nixpkgs-fmt.